pylutron/__init__.py

```diff
@@ -803,6 +815,34 @@ def stop(self):
     self._lutron.send(Lutron.OP_EXECUTE, Output._CMD_TYPE, self._integration_id,
         Output._ACTION_STOP)
 
+class Motor(Output):
```

Comment:
This won't support simple raise/lower.

My suggestion is to make this inherit from Shade, and move the enums out of Output into this class

Reply:
I cannot understand why this won't support raise/lower. Can you eleabrate on this?
Regarding inherriting from Shade. I think this is wrong, as this is a Motor entity, which is different from a Lutron Shade, even though we will use the motor as a shade In HA.